### PR TITLE
Check the ngeo local files

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 1]
```diff
@@ -19,10 +19,15 @@ Version 2.0.0
 
     git rm -r lingua.cfg
 
-3. pyramid_closure include:
+3. In the `vars_{{package}}.yaml` file:
 
-   If your project's main function includes `config.include('pyramid_closure')`,
-   remove it, as c2cgeoportal now takes care of including `pyramid_closure`.
+   * Remove the `vars.checker.lang_files` to use the default value
+     that test the ngeo language files.
```
Comment (asaunier) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: "Remove the `vars.checker.lang_files` parameter"
test => tests

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 2]
```diff
@@ -19,10 +19,15 @@ Version 2.0.0
...
+
+4. In the `{{package}}/__init__.py`file:
```
Comment (asaunier) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: missing blank space before "file"

**Comment by sbrunner** [comment]
Updated :-)

**Comment by asaunier** [comment]
LGTM

**Pull request merged**